### ISSUE TITLE
Wrong Interpretation

### ISSUE THREAD

**Issue by Tortoise17** [SoccerNet/sn-spotting#4]
I have to ask one question. I used NetVlad++ model and TempPooling inference results on my own data video test is that accuracy is less than CALF model json. All the events are less than value 0,25 in visibility.  Could you give any hint why it could be. I guess that I need correct K_V2 values

**Comment by Tortoise17** [comment]
@SilvioGiancola I have to ask for TempPooling in config. 
Like in CALF is 
```
# Values of the K parameters (in seconds) in the context-aware loss
K_V2 = torch.FloatTensor([[-100, -98, -20, -40, -96, -5, -8, -93, -99, -31, -75, -10, -97, -75, -20, -84, -18], [-50, -49, -10, -20, -48, -3, -4, -46, -50, -15, -37, -
                                                                                                                 5, -49, -38, -10, -42, -9], [50, 49, 60, 10, 48, 3, 4, 46, 50, 15, 37, 5, 49, 38, 10, 42, 9], [100, 98, 90, 20, 96, 5, 8, 93, 99, 31, 75, 10, 97, 75, 20, 84, 18]]).cuda()
```
Can I have values for NetVLAD++ ? I guess here I need your help. 
which values to be replaced for this model.

**Comment by SilvioGiancola** [comment]
Hi @Tortoise17 , You actually don't need `K_V2` for NetVLAD++. All you need to set is the size of the temporal window you want to pool from. That temporal value is generic for all the actions, my model/code cannot handle multiple values per class.

Now I don't know why the pre-trained model does not work as well as expected in your own video, and if CALF works better, maybe it means that CALF transfers better to your video, I can't tell.

**Comment by Tortoise17** [comment]
@SilvioGiancola Thankyou so much. Can you tell me as a guess about the temporal window size ? to decrease or increase?

**Comment by Tortoise17** [comment]
at the moment, it is 15

**Comment by SilvioGiancola** [comment]
The sensitivity analysis on SoccerNet was provided in the paper (Table 2): https://openaccess.thecvf.com/content/CVPR2021W/CVSports/papers/Giancola_Temporally-Aware_Feature_Pooling_for_Action_Spotting_in_Soccer_Broadcasts_CVPRW_2021_paper.pdf

**Comment by Tortoise17** [comment]
Thank you so much.
